azure-pipelines/build.yml

```diff
@@ -72,14 +72,16 @@ steps:
   inputs:
     command: test
     projects: test/Microsoft.SqlTools.ServiceLayer.UnitTests
-    arguments: '--logger "trx;LogFileName=xunit.trx"'
+    testRunTitle: SqlTools.ServiceLayer.UnitTests
+    arguments: '--configuration $(buildConfiguration) --collect "Code coverage"'
 
 - task: DotNetCoreCLI@2
   displayName: 'dotnet test test/Microsoft.Kusto.ServiceLayer.UnitTests'
   inputs:
     command: test
     projects: test/Microsoft.Kusto.ServiceLayer.UnitTests
-    arguments: '--logger "trx;LogFileName=xunit.trx"'
+    testRunTitle: Kusto.ServiceLayer.UnitTests
```

Comment:
testRunTitle sets the name of the test

![image](https://user-images.githubusercontent.com/63619224/135684693-9f28be11-502f-40b0-8ed7-d6aabe3bd0dd.png)